detox/jest.detox.startup.js

```diff
@@ -37,16 +37,16 @@ beforeAll(async () => {
             detoxURLBlacklistRegex: ".*firestore.*"
         },
         // JS actions
-        permissions: { faceid: "YES", location: "inuse", camera: "YES", photos: "YES" }
+        permissions: { faceid: "YES", location: "inuse", camera: "YES", photos: "YES", notifications: "YES" }
     });
 
     if (device.getPlatform() === "ios") {
         await prepDeveloperApp("localhost", 8080);
     }
     await waitFor(element(by.id("$screen")).atIndex(0))
         .toBeVisible()
-        .withTimeout(20000);
-}, 120000);
+        .withTimeout(180000);
```

Comment:
Why so long? That's really long

Reply:
Wesley said this made sense for local runs. I am fine with changing it back though.